packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -203,6 +203,52 @@ export default class MainProcess {
       this.windowsManager.forceFocusWindow();
     });
 
+    // Used in the `tsh install` command on macOS to make the bundled tsh available in PATH.
+    // Returns true if tsh got successfully installed, false if the user closed the osascript
+    // prompt. Throws an error when osascript fails.
+    ipcMain.handle('main-process-symlink-tsh', async () => {
```

Comment:
Since this is specific to macOS, should we indicate that in the name of the event?
```suggestion
    ipcMain.handle('main-process-symlink-tsh-macos', async () => {
```

Reply:
> Since this is specific to macOS, should we indicate that in the name of the event?

I was thinking about it but in the end I didn't do it for some reason, I'll add that.